usr/bin/cleanup

```diff
@@ -1,12 +1,6 @@
-#!/bin/bash
-
-# List of directories to clean
-DIRS=(
-    ~/.ssh
-    /root/loot
-)
+#!/bin/sh
```

Comment:
Doesn't make sense to get rid of the array here, it's easier to expand in the future.

Reply:
arrays aren't supported in posix shell. granted, this isn't the slowest of the scripts, but it still runs twice as fast in posix shell than it does in bash

Reply:
Fair enough, I actually think it's unlikely the list would be expanded in the future anyway, I agree it's worth getting rid of considering the speed advantage then.

Reply:
my concern was that this function looks like it does too much if you don't consider all the options (like switching mode and coming from an unknown state).  I wanted it clear why the function was so kill happy